roles/kubernetes-apps/ansible/templates/kubedns-deploy.yml

```diff
@@ -38,7 +38,7 @@ spec:
         livenessProbe:
           httpGet:
             path: /healthz
-            port: 8080
+            port: {{ kube_apiserver_insecure_port }}
```

Comment:
I don't think these ports should have changed -- kube-dns has a healthz endpoint independent of kube apiserver